sleap/gui/widgets/monitor.py

```diff
@@ -6,10 +6,11 @@
 import zmq
 import jsonpickle
 import logging
-from typing import Optional
+from typing import Optional, Dict
 from qtpy import QtCore, QtWidgets, QtGui
 from qtpy.QtCharts import QtCharts
 import attr
+from sleap.nn import training
```

Comment:
Let's definitely keep this separate. We should try to minimize coupling between backend and frontend, specially as we refactor the backend.

Can we move these ZMQ related functions to the frontend instead? Maybe under `gui/zmq.py` or `utils/zmq.py` or something?

Reply:
I have moved the functions to a separate new file under `gui/utils.py` where we could add any other GUI related utility functions in the future.